engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -367,7 +367,6 @@ public void testSelf1() throws Exception {
   }
 
   @Test
-  @Ignore
```

Comment:
This test shall now pass, thanks to the changes on the Rust side. CCing @kazcw .